frankenphp/docker-entrypoint.sh

```diff
@@ -1,14 +1,20 @@
 #!/bin/sh
 set -e
 
-# first arg is `-f` or `--some-option`
-if [ "${1#-}" != "$1" ]; then
-	set -- php-fpm "$@"
-fi
-
-if [ "$1" = 'php-fpm' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ]; then
+if [ "$1" = 'frankenphp' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ]; then
+	setfacl -R -m u:www-data:rwX -m u:"$(whoami)":rwX config
+	setfacl -dR -m u:www-data:rwX -m u:"$(whoami)":rwX config
+	setfacl -R -m u:www-data:rwX -m u:"$(whoami)":rwX composer.json
+	setfacl -dR -m u:www-data:rwX -m u:"$(whoami)":rwX composer.json
+	setfacl -R -m u:www-data:rwX -m u:"$(whoami)":rwX composer.lock
+	setfacl -dR -m u:www-data:rwX -m u:"$(whoami)":rwX composer.lock
+	setfacl -R -m u:www-data:rwX -m u:"$(whoami)":rwX symfony.lock
+	setfacl -dR -m u:www-data:rwX -m u:"$(whoami)":rwX symfony.lock
 
-  echo Running $1...
+	#if [ -z "$(ls -A 'vendor/' 2>/dev/null)" ]; then
+	#	composer install --prefer-dist --no-progress --no-interaction
+	#fi
```

Comment:
copied this from Symfony docker, might be interesting to only call "bash install.sh" if the vendor folder does not exist ... not sure yet